bluetooth_mesh/provisioning.py

```diff
@@ -27,6 +27,7 @@ class ProvisioningPDUType(enum.IntEnum):
     DATA = 0x07
     COMPLETE = 0x08
     FAILED = 0x09
+    ACK = 0xFF
```

Comment:
Since this never goes into any payload field and `0xff` PDU type is not defined in the spec, set this to `-1` and add a comment.

Reply:
Changed